src/components/core/RightCol.vue

```diff
@@ -35,14 +32,12 @@
       </div>
     </div>
   </div>
-  <!-- fin colonne de droite -->
 </template>
 
 <script lang="ts">
   import { Options, Vue } from 'vue-class-component';
   import OperationsSelector from "../rightCol/OperationsSelector.vue";
   import ThisWeek from "../rightCol/ThisWeek.vue";
-  //import LastWeek from "../rightCol/LastWeek.vue";
   import AllTrs from "../rightCol/AllTrs.vue";
   import MyModal from "../modal/MyModal.vue";
```

Comment:
still needed ?

Reply:
I'm not sure I get what you mean ?
You're suggesting to remove also the `rightCol/ThisWeek.vue` sub-component ?

Reply:
Oh, no, you're suggesting to remove the MyModal stanza, right ?

Reply:
Yep, I thought the line of code would be highlighted correctly, it seems it wasn't. Yes I confirm that I was talking about `MyModal` component seemed to be removeable at first glance, I didn't know if you wanted to keep it for some future work on the file or not. This is why I usually formulate my remarks as questions.